### PR TITLE
Implement getBlockExplorerURL

### PR DESCRIPTION
**Pull request by KadenZipfel**
Closes: https://github.com/protofire/omen-exchange/issues/1703
Depends on: https://github.com/protofire/omen-exchange/pull/1741

### CODE REVIEW

File: app/src/util/tools.ts [hunk 1]
```diff
@@ -758,7 +758,7 @@ export const reverseArray = (array: any[]): any[] => {
   return newArray
 }
 
-export const getBlockExplorerURL = (networkId: number, txHash: string): string => {
+export const getTxHashBlockExplorerURL = (networkId: number, txHash: string): string => {
```
Comment (Mi-Lan) [inline app/src/util/tools.ts:761]: Don't know if it's scope of this issue but i would add a suggestion that all etherscan links are stored in networks and that we make a function that fetches it based on network like we did for bunch of stuff....like for example getChainSpecifiAlternativeUrls? @KadenZipfel @hexyls what do you think?

**Review by Mi-Lan — COMMENTED** [review]
Will give it another round of review after it's merged with master...Theres bunch of left out code

File: app/src/util/tools.ts (hunk 1 above)
Comment (KadenZipfel) [reply app/src/util/tools.ts]: Good call! I've moved it to the networks file

**Review by Mi-Lan — COMMENTED** [review]
Should we remove EtherscanLink enum from types.ts?

**Review by Mi-Lan — APPROVED** [review]
Other than that small thing should be GTG !!!!!

**Comment by KadenZipfel** [comment]
> Should we remove EtherscanLink enum from types.ts? Seems that it's unused now

Ahhh, yes of course! Removed

**Pull request merged**